lib/mockexpress.js

```diff
@@ -51,6 +63,10 @@ var MockExpress = function(route) {
 				return getMap[property];
 			};
 
+      if (requestObject.get('body')) {
```

Comment:
This special case should not be necessary. The module doesn't enforce any rules on req (apart from `.query`, `.method`, `.url` as defined in the `invoke` function above. You can simply set `req.body` before calling `invoke`, ref - test below.